Makefile.js

```diff
@@ -298,11 +297,19 @@ target.gensite = function() {
 
             var baseName = path.basename(filename);
             var sourceBaseName = path.basename(filename, ".md") + ".js";
+            var ruleName = path.basename(filename, ".md");
 
-            text = "---\ntitle: ESLint\nlayout: doc\n---\n<!-- Note: No pull requests accepted for this file. See README.md in the root directory for details. -->\n" + text;
+            // 4. Prepend page title and layout variables at the top of rules
+            if (path.dirname(filename).indexOf("rules") >= 0) {
+                text = "---\ntitle: " + (ruleName === "README" ? "List of available rules" : "Rule " + ruleName) + "\nlayout: doc\n---\n<!-- Note: No pull requests accepted for this file. See README.md in the root directory for details. -->\n" + text;
+            } else {
+                text = "---\ntitle: Documentation\nlayout: doc\n---\n<!-- Note: No pull requests accepted for this file. See README.md in the root directory for details. -->\n" + text;
+            }
 
+            // 5. Change links extension from md to html
             text = text.replace(/\.md(.*\))/g, ".html$1").replace("README.html", "index.html");
```

Comment:
It does. Easy test: 

http://eslint.org/docs/rules/comma-dangle.html
http://eslint.org/docs/rules/comma-dangle